storage/storage_interface.ml

```diff
@@ -118,14 +119,17 @@ type dp_stat_t = {
 
 let string_of_dp_stat_t (x: dp_stat_t) = Jsonrpc.to_string (rpc_of_dp_stat_t x)
 
+type device_config = (string * string) list
```

Comment:
This is not used here.

Reply:
It was used in an earlier commit, I've removed it now.
There are quite a few `device_config: (string * string) list` parameters, but it is not worth introducing a type alias for this, since we need to convert this interface to PPX anyway.